src/Rings/MPolyQuo.jl

```diff
@@ -505,7 +505,13 @@ parent(a::MPolyQuoRingElem) = a.P
 
 zero(R::MPolyQuoRing) = MPolyQuoRingElem(zero(base_ring(R)), R, true)
 
-iszero(a::MPolyQuoRingElem) = iszero(simplify(a).f)
+iszero(a::MPolyQuoRingElem{T}) where {S<:Union{FieldElem, ZZRingElem}, T<:MPolyElem{S}} = iszero(simplify(a).f)
+iszero(a::MPolyQuoRingElem{<:MPolyDecRingElem{<:FieldElem}}) = iszero(simplify(a).f)
+
+function is_zero(a::MPolyQuoRingElem)
+  return iszero(simplify(a).f)
```

Comment:
```suggestion
  return a.f in modulus(parent(a))
```
or something like that?